src/main/webapp/scripts/script.js

```diff
@@ -82,6 +82,38 @@ function parseTime(time){
   return timeString;
 }
 
+/**
+ * Function to prove check if business is still opened if it hasn't been updated
+ * @param {Object} saleCard - Salecard object that contains its info
+ * @return {boolean} 
```

Comment:
Can you add here what does this return variable stands for ?

Reply:
For sure!